src/utils/deepLink.js

```diff
@@ -11,14 +11,17 @@ import Navigation from '@/utils/navigation';
 
 function handleWalletConnect(uri, isUnlocked) {
   const { dispatch } = store;
-  dispatch(walletConnectSetPendingRedirect());
+  const { query } = new URL(uri);
+  const { bridge, requestId } = qs.parse(query.substring(1));
```

Comment:
does this `qs.parse(query.substring(1));` always has `{ bridge, requestId }`?

Reply:
Not always but it is the idea, if  bridge is undefined it is not a firstConnection uri